controllers/apply/expiries/views/awards-for-all-expiry-email.njk

```diff
@@ -1,45 +1,45 @@
 {% extends "../../../../views/layouts/emails.njk" %}
 
 {% block content %}
-
     {% if isBilingual %}
-        <p>
-            <strong>
-                Bilingual message – please scroll down for Welsh.<br />
-                Neges ddwyieithog – y Gymraeg i’w weld isod.
-            </strong>
-        </p>
+        <p><strong>
+            Bilingual message – please scroll down for Welsh.<br />
+            Neges ddwyieithog – y Gymraeg i’w weld isod.
+        </strong></p>
     {% endif %}
-
     <p>Hello,</p>
-
     <p>
-        <strong>Your National Lottery Awards for All application isn’t finished yet.</strong><br/>
-        We noticed you started applying{% if projectName %} for {{ projectName | safe }}{% endif %}. But you haven’t finished your application.
+        <strong>Your National Lottery Awards for All application isn’t finished yet</strong><br/>
+        We noticed you started applying{% if projectName %} for {{ projectName | safe }}{% endif %},
+        but haven’t finished your application.
     </p>
-
     <p>
-        <strong>You have until {{ expiryDate.en }} to finish your application.</strong><br/>
-        If you don’t complete your application before that time, we’ll delete it - so that we can keep your data safe.
+        <strong>
+            You have until {{ expiryDate.en }}
+            to finish your application
+        </strong><br/>
+        If you don’t complete it at that time,
```

Comment:
Should this say "before that time"?

Reply:
Good shout, this is what we had before, I've updated this to keep that phrasing.